integration-tests/deployment/ccip/state.go

```diff
@@ -40,10 +41,11 @@ type CCIPChainState struct {
 	OnRamp             *onramp.OnRamp
 	OffRamp            *offramp.OffRamp
 	FeeQuoter          *fee_quoter.FeeQuoter
-	ArmProxy           *rmn_proxy_contract.RMNProxyContract
+	RMNProxy           *rmn_proxy_contract.RMNProxyContract
```

Comment:
should we be making an effort to use `RMN` instead of `ARM` everywhere from now on? I still see them used pretty interchangeably